app/static/css/main.css

```diff
@@ -122,3 +122,27 @@ textarea {
         position: static;
     }
 }
+
+.scrollTopBtn {
+    border: none;
+    border-radius: 50%;
+    color: white;
+    background-color: #00356b;
+    cursor: pointer;
+    font-size: 30px;
+    text-align: center;
+    line-height: 38px;
+    padding-bottom: 15px;
+    width: 60px;
```

Comment:
Can you also specify a height? It looks like it's slightly oval right now, it should be evenly circle